src/test/java/uk/gov/mca/beacons/service/BeaconsRegistrationServiceUnitTest.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.mca.beacons.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class BeaconsRegistrationServiceUnitTest {
+
+  @Test
+  void pretendImTests() {
+    assertEquals(true, true);
```

Comment:
@Ninamma I would just import the `BeaconsRegistrationService` class here and call the static `main` method for now.  And put a `TODO` before the test so that it is picked up by IDEs and we can remove it once we have other unit tests inplace